entries/q/qtrade.ca.json

```diff
@@ -0,0 +1,17 @@
+{
+  "Qtrade": {
+    "domain": "qtrade.ca",
+    "url": "https://www.qtrade.ca",
```

Comment:
If you go to `https://<domain>` and it redirects to the value of the `url`-element then the `url`-element isn't needed.
Since this is the case for this PR, please remove the `url`-element:
```suggestion
```